src/scripts/site.js

```diff
@@ -420,14 +420,14 @@ var AppInfo = {};
                 return self.ResizeObserver;
             });
         } else {
-            define("ResizeObserver", ["bower_components/emby-webcomponents/resize-observer-polyfill/ResizeObserver"], returnFirstDependency);
+            define("ResizeObserver", ["components/resize-observer-polyfill/ResizeObserver"], returnFirstDependency);
         }
     }
 
     function initRequireWithBrowser(browser) {
         var bowerPath = getBowerPath();
```

Comment:
can we remove or rename these (both variable and function)?

Reply:
Well the objective is that when webpack conversion is done, there will be no components inside bower_components. And if still remain some, it will be properly renamed.